packages/gitbeaker-requester-utils/src/RequesterUtils.ts

```diff
@@ -110,6 +110,7 @@ export function modifyServices<T>(services: T, customConfig: Record<string, unkn
 
   Object.entries(services).forEach(([k, s]) => {
     updated[k] = extendClass(s, customConfig);
+    if (k === 'GroupMembers') console.log(updated[k]);
```

Comment:
Hey @jdalrymple, was this line added by mistake? I noticed `gitbreaker` started printing out for some reason, at it originated for here

Reply:
Oh oops! Yaa ill remove it.